app/assets/javascripts/discourse/app/lib/sidebar/admin-sidebar.js

```diff
@@ -59,9 +60,26 @@ class SidebarAdminSectionLink extends BaseCustomSidebarSectionLink {
   get title() {
     return this.adminSidebarNavLink.text;
   }
+
+  get currentWhen() {
+    // NOTE: This is weird, but since our admin plugin routes starting with adminPlugins.
+    // are nested beneath /admin/plugins, Ember tries to highlight both the Installed Plugins
+    // and relevant plugin link.
+    //
+    // We only want to highlight the top level Installed Plugins link if we are on that page,
+    // not if we are on e.g. the chat plugin admin route.
+    if (
+      this.route === "adminPlugins.index" &&
+      this.router.currentRoute.name === "adminPlugins.index"
+    ) {
+      return "adminPlugins.index";
+    } else {
+      return this.route;
+    }
+  }
```

Comment:
I'm not sure if this is still required because I removed this locally and got this:

<img width="272" alt="Screenshot 2024-02-23 at 7 26 36 AM" src="https://github.com/discourse/discourse/assets/4335742/b583b158-b23e-4fb6-bae4-48c0f551461d">